README.md

```diff
@@ -868,6 +868,30 @@ Installs SQL Server on the target node.
 * **[String[]]FailoverClusterIPAddress** _(Write)_: Array of IP Addresses to be assigned to the clustered SQL Server instance. IP addresses must be in [dotted-decimal notation](https://en.wikipedia.org/wiki/Dot-decimal_notation), for example ````10.0.0.100````. If no IP address is specified, uses 'DEFAULT' for this setup parameter.
 * **[String] FailoverClusterNetworkName** _(Write)_: Host name to be assigned to the clustered SQL Server instance.
 
+##### Required Parameters
+
+For configurations that utilize the 'InstallFailoverCluster' action, the following parameters are required (beyond those required for the standalone installation:
+- InstanceName (can be MSSQLSERVER if you want to install a default clustered instance)
```

Comment:
Huh, I actually have that extension installed. But it's not showing any problems. Weird... I'll work on this, shouldn't be too hard to fix.